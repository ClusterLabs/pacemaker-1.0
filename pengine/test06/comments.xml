<?xml version="1.0"?>
<!DOCTYPE cib SYSTEM "crm-1.0.dtd">
<!-- initial -->
<cib num_updates="1" epoch="1" admin_epoch="0">
  <configuration>
    <!-- pre-comment -->
    <crm_config><cluster_property_set id="cib-bootstrap-options"><attributes>
      <nvpair id="no_quorum_policy" name="no_quorum_policy" value="ignore"/>
      <!-- post-comment -->
    </attributes></cluster_property_set></crm_config>
    <nodes>
      <node id="node2" uname="node2" type="member"/>
      <!-- embedded-comment -->
      <node id="node1" uname="node1" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc1" class="heartbeat" type="apache">
        <instance_attributes id="any">
          <attributes>
            <nvpair  id="1" name="priority" value="1.0"/>
          </attributes>
        </instance_attributes>
      </primitive>
      <primitive id="rsc2" class="heartbeat" type="apache">
        <instance_attributes id="any">
          <attributes>
            <nvpair  id="2" name="priority" value="1.0"/>
          </attributes>
        </instance_attributes>
      </primitive>
    </resources>
    <constraints>
      <rsc_location id="run_rsc1" rsc="rsc1">
	<rule id="cannot1" score="-INFINITY" boolean_op="or"><!-- no spaces --><expression id="exp1" attribute="#uname" operation="eq" value="node2"/>
	</rule>
      </rsc_location>
    </constraints>
  </configuration>
  <status>
    <node_state id="node1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true"/>
    <node_state id="node2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true"/>
  </status>
</cib>
