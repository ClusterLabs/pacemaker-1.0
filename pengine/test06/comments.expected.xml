<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="1" dc-uuid="0" have-quorum="false" remote-tls-port="0" validate-with="pacemaker-1.0">
  <configuration>
    
    <crm_config><cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/></cluster_property_set></crm_config>
    <nodes>
      <node id="node2" uname="node2" type="member"/>
      
      <node id="node1" uname="node1" type="member"/>
    </nodes>
    <resources>
      <primitive id="rsc1" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc1.meta"/>
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="priority" value="1.0"/></instance_attributes>
      </primitive>
      <primitive id="rsc2" class="heartbeat" type="apache"><meta_attributes id="primitive-rsc2.meta"/>
        <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="priority" value="1.0"/></instance_attributes>
      </primitive>
    </resources>
    <constraints>
      <rsc_location id="run_rsc1" rsc="rsc1">
	<rule id="cannot1" score="-INFINITY" boolean-op="or"><expression id="expression" attribute="#uname" operation="eq" value="node2"/>
	</rule>
      </rsc_location>
    </constraints>
  </configuration>
  <status>
    <node_state id="node1" ha="active" uname="node1" crmd="online" join="member" expected="member" in_ccm="true"/>
    <node_state id="node2" ha="active" uname="node2" crmd="online" join="member" expected="member" in_ccm="true"/>
  </status>
</cib>
